repos/fdbt-site/src/pages/products/productDetails.tsx

```diff
@@ -473,6 +473,35 @@ const createProductDetails = async (
         });
     }
 
+    if ('pricingByDistance' in product && product.pricingByDistance) {
+        const pricePerDistance = product.pricingByDistance;
+
+        productDetailsElements.push({
+            id: 'pricing-by-distance-price',
+            name: 'Prices',
+            content: [`Min price - £${pricePerDistance.minimumPrice}`, `Max price - £${pricePerDistance.maximumPrice}`],
+            editLink: '/definePricingPerDistance',
+        });
+
+        const distanceBands: string[] = [];
+        pricePerDistance.capPricing.forEach((capDistance, index) => {
```

Comment:
Why are we pushing data to one array, only to then map through that array and move it all to another array?

Can we not just get rid of the distanceBands array and push straight into the productDetailsElements array?